rust/processor/src/processors/fungible_asset_processor.rs

```diff
@@ -159,9 +159,19 @@ fn insert_fungible_asset_metadata_query(
             .do_update()
             .set(
                 (
+                    creator_address.eq(excluded(creator_address)),
+                    name.eq(excluded(name)),
+                    symbol.eq(excluded(symbol)),
                     supply_v2.eq(excluded(supply_v2)),
                     maximum_v2.eq(excluded(maximum_v2)),
                     decimals.eq(excluded(decimals)),
```

Comment:
these should be at the end. same ordering as the schema I believe.

Reply:
sure